index.js

```diff
@@ -121,8 +121,13 @@ module.exports = results => {
 
 		if (x.type === 'message') {
 			let ruleUrl;
+
 			try {
-				ruleUrl = getRuleDocs(x.ruleId).url;
+				if (typeof data === 'undefined') {
```

Comment:
This check is not correct. `data` can exist even without the `rulesMeta.url` property being available. You need to check for the native `.url` property first and then fall back to `getRuleDocs`. This also needs docs and tests.

---

This issue has a bounty, so it's expected that you put a great effort into the PR.

Reply:
Actually we are falling back to same solution, thus same tests will do, isn't it?